examples/api/test/widgets/dismissible/dismissible.0_test.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/dismissible/dismissible.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets(
+    'ListTiles can be dismissed from right to left',
+    (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const example.DismissibleExampleApp(),
+      );
+
+      for (final int index in <int>[0, 33, 66, 99]) {
+        final ValueKey<int> key = ValueKey<int>(index);
+
+        await tester.scrollUntilVisible(find.byKey(key), 100);
+
+        expect(find.byKey(key), findsOneWidget);
+
+        await tester.dismissHorizontally(
+          find.byKey(key),
+          direction: AxisDirection.left,
+        );
+
+        await tester.pumpAndSettle();
+
+        expect(find.byKey(key), findsNothing);
+      }
+    },
+  );
+
+  testWidgets(
+    'ListTiles can be dismissed from left to right',
+    (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const example.DismissibleExampleApp(),
+      );
+
+      for (final int index in <int>[0, 33, 66, 99]) {
+        final ValueKey<int> key = ValueKey<int>(index);
+
+        await tester.scrollUntilVisible(find.byKey(key), 100);
+
+        expect(find.byKey(key), findsOneWidget);
+
+        await tester.dismissHorizontally(
+          find.byKey(key),
+          direction: AxisDirection.right,
+        );
+
+        await tester.pumpAndSettle();
+
+        expect(find.byKey(key), findsNothing);
+      }
+    },
+  );
+}
+
+extension on WidgetTester {
```

Comment:
The styleguide says to avoid `extension`: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-extension

I would just do this as a function here.

Reply:
Thanks for the review!
Replaced extension by function as you suggested.